README.md

```diff
@@ -352,3 +353,38 @@ VA7:     !include configs/env=int/region=va7/kafka-brokers.yaml regionBrokers.VA
 ```
 
 This will replace the value after interpolation with the value of the regionBrokers.VA7 found under the configs/env=int/region=va7/kafka-brokers.yaml path.
+
+## custom merge strategy
```

Comment:
Minor: To be consistent with the other sections, let's start the section name with a capital letter: `Custom merge strategy`

Reply:
Done.